### PR TITLE
remade lesson-2 homework, PR didnt work 20.24 31.08.2021

### PR DESCRIPTION
**Pull request by olennikovandrey**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
```
Comment (Dronom) [inline .eslintrc.js:4]: add node: true

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+let user = 'John Doe';
+console.log(user);
+
+let student = 'Olennikov Andrey'
+console.log(student);
+
+if (true) {
+    user = student; //it will be student's value;
+}
```
Comment (Dronom) [inline js/main.js:9]: Why do use use if here?

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+for (let i = 0; i < point9.length; i++) {
+    if (point9[i] > 5 && point9[i] < 10) {
+        console.log(point9[i]);
+    }
+}
+
+//point 10 of homework
+let point10 = [2, 5, 8, 15, 0, 6, 20, 3];
+for (let i = 0; i < point10.length; i++) {
+    if (point10[i] % 2 == 0) {
```
Comment (Dronom) [inline js/main.js:42]: use === instead of ==

Comment (olennikovandrey) [reply js/main.js:42]: fixed

File: .eslintrc.js (hunk 1 above)
Comment (olennikovandrey) [reply .eslintrc.js:4]: done

File: js/main.js (hunk 2 above)
Comment (olennikovandrey) [reply js/main.js:9]: idiot))) fixed)

File: .eslintrc.js (hunk 1 above)
Comment (olennikovandrey) [reply .eslintrc.js:4]: done

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+console.log(test); //the result is 20, because the numerical value "1" is subtracted from the value of the first digit "2", and the string value "1" will remain unchanged
+
+test = Boolean(test); //the result is "true"
+console.log(test);
+
+//point 8 of homework
+let multiplication = 1;
+let point8 = [2, 3, 5, 8];
+for (let i = 0; i < point8.length; i++) {
+	multiplication *= point8[i];
```
Comment (Dronom) [inline js/main.js:27]: don't name your variable like point8

**Review by Dronom — COMMENTED** [review]
all is working, but it will be better if you name variables to understand what they do

File: js/main.js (hunk 4 above)
Comment (olennikovandrey) [reply js/main.js:27]: changed point8, -9, -10 to first-, second, third_arr

**Pull request closed without merging**